src/main/java/uk/ac/ebi/eva/commons/models/converters/data/VariantStorageManager.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2015-2016 OpenCB
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.commons.models.converters.data;
+
+public abstract class VariantStorageManager {
+
+    public enum IncludeSrc {
+        NO, FIRST_8_COLUMNS, FULL
```

Comment:
We want to use always `FIRST_8_COLUMNS` and the pipeline code has already been modified to do so. The converters interface would be heavily simplified if that value was always assumed.